chassis_test.go

```diff
@@ -23,17 +23,19 @@ import (
 const (
 	CHASSIS_HOSTNAME = "fakehost"
 	CHASSIS_NAME     = "fakechassis"
-	ENCAP_TYPE       = "geneve"
 	IP               = "10.0.0.11"
 )
 
+// can be one or many encap_types similar to chassis-add of sbctl
+var ENCAP_TYPES = []string{ "stt", "geneve", "vxlan" }
+
 func TestChassis(t *testing.T) {
 	db = getOVNDB()
 	if db == dbNB {
 		t.Skip("Skip running chassis test against nb db")
 	}
 	t.Logf("Adding Chassis to OVN SB DB")
-	ocmd, err := ovndbapi.ChassisAdd(CHASSIS_NAME, CHASSIS_HOSTNAME, ENCAP_TYPE, IP)
+	ocmd, err := ovndbapi.ChassisAdd(CHASSIS_NAME, CHASSIS_HOSTNAME, ENCAP_TYPES, IP)
 	if err != nil {
 		t.Fatal(err)
 	}
```

Comment:
Please ensure Encaps are written to DB and can be read back correctly. I didn't see implementation of populating encaps back, so there is no way for the current implementation to read encaps, and other fields such as external_ids, transport_zones, etc. of Chassis record.

Reply:
Sure will add chassis list and encap list support in revised patch.